include/cantera/kinetics/Falloff.h

```diff
@@ -258,6 +258,65 @@ class SRI : public Falloff
     doublereal m_e;
 };
 
+//! The 1- or 2-parameter Tsang falloff parameterization.
+/*!
+ *  The Tsang falloff model is adapted from that of Troe.
+ *  It provides a constant or linear in temperature value for \f$ F_{cent} \f$:
+ *  \f[ F_{cent} = A + B*T \f]
+ *  
+ *  The value of \f$ F_{cent} \f$ is then applied to Troe's model for the
+ *  determination of the value of \f$ F \f$.
+ *
+ * @ingroup falloffGroup
+ */
+class Tsang : public Falloff
+{
+public:
+    //! Constructor
+    Tsang() : m_a(0.0), m_b(0.0) {}
+
+    //! Initialization of the object
+    /*!
+     * @param c Vector of one or two doubles: The doubles are the parameters,
+     *          a and (optionally) b of the Tsang F_cent parameterization
+     */
+    virtual void init(const vector_fp& c);
+
+    //! Update the temperature parameters in the representation
+    /*!
+     *   @param T         Temperature (Kelvin)
+     *   @param work      Vector of working space, length 1, representing the
+     *                    temperature-dependent part of the parameterization.
+     */
+    virtual void updateTemp(doublereal T, doublereal* work) const;
+
+    virtual doublereal F(doublereal pr, const doublereal* work) const;
```

Comment:
No `doublereal` in new code (just use `double`) ... here and elsewhere.

Reply:
Made this change throughout file (my use of `doublereal` was for consistency with other functions)